src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala

```diff
@@ -93,7 +93,8 @@ class JdbcReadJournal(config: Config)(implicit val system: ExtendedActorSystem)
 
   // Started lazily to prevent the actor for querying the db if no eventsByTag queries are used
   private[query] lazy val journalSequenceActor = system.actorOf(
-    JournalSequenceActor.props(readJournalDao, readJournalConfig.journalSequenceRetrievalConfiguration), "akka-persistence-jdbc-journal-sequence-actor")
+    JournalSequenceActor.props(readJournalDao, readJournalConfig.journalSequenceRetrievalConfiguration), "akka-persistence-jdbc-journal-sequence-actor"
+  )
```

Comment:
Scalariform is forcing an annoying formatting here by putting the ) on the next line. I have had it as well on the PR I have openned. 

Maybe the best here is to format it as:
```scala
private[query] lazy val journalSequenceActor = 
  system.actorOf(
    JournalSequenceActor.props(readJournalDao, readJournalConfig.journalSequenceRetrievalConfiguration),
    "akka-persistence-jdbc-journal-sequence-actor"
  )
```

As such, we can at least have this trailing ) on the same column as `system.actor` which where the open ( comes from.

Reply:
Good point